streams-components/streams-http/src/main/java/org/apache/streams/components/http/persist/SimpleHTTPPostPersistWriter.java

```diff
@@ -113,13 +113,14 @@ protected ObjectNode preparePayload(StreamsDatum entry) {
     public HttpPost prepareHttpPost(URI uri, ObjectNode payload) {
         HttpPost httppost = new HttpPost(uri);
         httppost.addHeader("content-type", this.configuration.getContentType());
+        httppost.addHeader("accept-charset", "UTF-8");
         try {
             String entity = mapper.writeValueAsString(payload);
             httppost.setEntity(new StringEntity(entity));
         } catch (JsonProcessingException e) {
-            e.printStackTrace();
+            LOGGER.warn(e.getMessage());
```

Comment:
i've typically used WARN when a specific datum fails to process/persist, but there's no evidence that all future data will fail.